dbt/include/bigquery/macros/relations/materialized_view/refresh.sql

```diff
@@ -0,0 +1,10 @@
+{% macro bigquery__refresh_materialized_view(relation) %}
+    get_replace_materialized_view_as_sql(
```

Comment:
I probably mentioned to do this initially, but let's replace this with the correct ddl.